kubernetes/windows/main.ps1

```diff
@@ -607,12 +655,18 @@ if (![string]::IsNullOrEmpty($requiresCertBootstrap) -and `
 $isAADMSIAuth = [System.Environment]::GetEnvironmentVariable("USING_AAD_MSI_AUTH")
 if (![string]::IsNullOrEmpty($isAADMSIAuth) -and $isAADMSIAuth.ToLower() -eq 'true') {
     Write-Host "skipping agent onboarding via cert since AAD MSI Auth configured"
+
+    #start Windows AMA 
+    Start-Job -ScriptBlock { Start-Process -NoNewWindow -FilePath "C:\opt\windowsazuremonitoragent\windowsazuremonitoragent\Monitoring\Agent\MonAgentLauncher.exe" -ArgumentList @("-useenv")}
```

Comment:
lets shorten this path by removing the redundant "windowsazuremonitoragent" directory.

Reply:
There are two directories under C:\opt\windowsazuremonitoragent - 
windowsazuremonitoragent which has the actual bits of AMA 
datadirectory which has all the output files for AMA